don't have res_nquery in musl, so use the macos res_query fallback
--
--- a/netwerk/dns/PlatformDNSUnix.cpp
+++ b/netwerk/dns/PlatformDNSUnix.cpp
@@ -32,11 +32,11 @@
   rv = packet.FillBuffer(
       [&](unsigned char response[DNSPacket::MAX_SIZE]) -> int {
         int len = 0;
-#if defined(XP_LINUX)
+#if 0 && defined(XP_LINUX)
         len = res_nquery(&_res, host.get(), ns_c_in,
                          nsIDNSService::RESOLVE_TYPE_HTTPSSVC, response,
                          DNSPacket::MAX_SIZE);
-#elif defined(XP_MACOSX)
+#elif 1 || defined(XP_MACOSX)
         len =
             res_query(host.get(), ns_c_in, nsIDNSService::RESOLVE_TYPE_HTTPSSVC,
                       response, DNSPacket::MAX_SIZE);
